.github/workflows/ci-main.yml

```diff
@@ -212,15 +120,9 @@ jobs:
           if [[ "$COV_RUN" == "true" ]]
           then
             echo "Running Coverage Test"
-            . test.env && poetry run pytest -n auto tests tools \
-              --cov c7n --cov tools/c7n_azure/c7n_azure \
-              --cov tools/c7n_gcp/c7n_gcp --cov tools/c7n_kube/c7n_kube \
-              --cov tools/c7n_tencentcloud/c7n_tencentcloud \
-              --cov tools/c7n_left/c7n_left \
-              --cov tools/c7n_mailer/c7n_mailer
-            poetry run coverage xml
+            make test-coverage COVERAGE_TYPE=xml
```

Comment:
the only real difference in coverage  configuration is going to a separate coverage xml build process via the native tool instead of via the pytest-coverage plugin, that might explain the coverage delta, albeit seems like a bug in pytest-coverage plugin if so.

Reply:
super weird, but yeah doing a separate coverage xml generation did resolve it.